test/cmd/quota.sh

```diff
@@ -14,6 +14,7 @@ os::cmd::try_until_text 'oc describe appliedclusterresourcequota/for-deads -n fo
 
 
 os::cmd::expect_success 'oc create clusterquota for-deads-by-annotation --project-annotation-selector=openshift.io/requester=deads --hard=secrets=50'
+os::cmd::expect_success 'oc create clusterquota for-deads-email-by-annotation --project-annotation-selector=openshift.io/requester=deads@deads.io --hard=secrets=50'
```

Comment:
add a couple negative tests:
* bad annotation key
* non-key=value format

add a positive test that exercises a selector containing a value with a comma